src/GitHubCreateMergePRs/config.xml

```diff
@@ -11,17 +11,14 @@
 
     <!-- Roslyn non-vs-deps branches (flowing between releases) -->
     <merge from="release/dev16.10" to="main" />
-    <merge from="main" to="release/dev17.0" />
 
     <!-- Roslyn vs-deps branches (flowing between releases) -->
     <merge from="release/dev16.10-vs-deps" to="main-vs-deps" />
-    <merge from="main-vs-deps" to="release/dev17.0-vs-deps" />
-    <merge from="release/dev17.0-preview1-vs-deps" to="release/dev17.0-vs-deps" />      
+    <merge from="release/dev17.0-preview1-vs-deps" to="main-vs-deps" />      
```

Comment:
theoretically we should be flowing from release/dev16.10-vs-deps -> release/dev17.0-preview1-vs-deps -> main-vs-deps, but we're not ready to take changes in release/dev17.0-preview1-vs-deps, right @genlu ?

Reply:
Yes, we'd like to wait a couple more days until the interop work is considered done, should be able to resume by the end of this week. So maybe we can start creating the merge PR and fixing potential conflicts before then?